README.md

```diff
@@ -64,24 +66,25 @@ In case you don't want to use the pre-compiled version, you can build the librar
 
 Prerequisites:
 
- * Download/the [**JDK 7**](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html) or [**JDK 8**](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) from Oracle and install it.
+ * Download [**JDK 7**](http://www.oracle.com/technetwork/java/javase/downloads/jdk7-downloads-1880260.html) or [**JDK 8**](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and install it.
  * Download & install the Android SDK, **Android 7.1 (API 25)** (for example through Android Studio’s **Android SDK Manager**)
 
-Once you have completed all the pre-requisites building Realm is done with a simple command
+Once you have completed all the pre-requisites building the adapter library is done with a single command:
 
 ```
 ./gradlew assemble
 ```
 
 That command will generate:
 
- * an aar file for the adapter library in `adapters/build/outputs/aar/android-adapters-release.aar`
+ * an `aar` file for the adapter library (`adapters/build/outputs/aar/android-adapters-release.aar`)
 
 ### Other Commands
 
  * `./gradlew monkeyDebug` will run the monkey tests on the example project.
  * `./gradlew javadoc` will create the javadoc for the library.
- * `./gradlew publishToSonatype` will upload artifacts to Maven Central.
+ * `./gradlew artifactoryPublish` will upload a SNAPSHOT to OJO.
```

Comment:
These are being deleted. When we move to Maven Central, the `publishToSonatype` was correct.

Reply:
```suggestion
 * `./gradlew publishToSonatype` will upload artifacts to Maven Central.
```
Thanks, fixing.